tests/testcore/onebox.go

```diff
@@ -541,9 +540,6 @@ func (c *TemporalImpl) startWorker() {
 		CurrentClusterName:       c.clusterMetadataConfig.CurrentClusterName,
 		ClusterInformation:       maps.Clone(c.clusterMetadataConfig.ClusterInformation),
 	}
-	if c.workerConfig.EnableReplicator {
-		clusterConfigCopy.EnableGlobalNamespace = true
```

Comment:
So this has no actual effect then?

Reply:
For non-xdc functional tests we don't need global namespace support: they all are for single cluster. I am setting this explicitly in `xdcBaseSuite` and actually, I see no harm enabling it always. I will probably do it later.